tests/antispoof/tests.c

```diff
@@ -266,6 +266,23 @@ static void test_antispoof_generic(const unsigned char **pkts,
 	pg_antispoof_arp_del_all(antispoof); 
 	REPLAY(0);
 
+	/* add ARP_MAX */
+       for (int i=0;i<=100;i++){
+	       if (i!=100){
+		       g_assert(!pg_antispoof_arp_add(antispoof, i, &error));
+                       g_assert(!error);
+	       }else {
+		       g_assert(pg_antispoof_arp_add(antispoof, i, &error));
+	       }
+       }	       
+       g_assert(error);
+       pg_error_free(error);
+       error = NULL;
```

Comment:
this idiom is use a lot in the test.
which make me think, that maybe an 
``` #define PG_ERROR_FREE(error) do {pg_error_free(error); error = NULL;} while(0);```
would be useful, (that in another commit)